gensim/models/base_any2vec.py

```diff
@@ -1297,20 +1338,31 @@ def _log_progress(self, job_queue, progress_queue, cur_epoch, example_count, tot
         always be equal to -1.
 
         """
+        if self.compute_loss:
```

Comment:
This feels redundant. Why is it here? How is this different from what is on line 1358?

Reply:
Yes absolutely, this should not be here, it is a mishap of my rebasing.